lib/web_ui/test/canvaskit/scene_test.dart

```diff
@@ -49,6 +49,17 @@ void testMain() {
       final ui.Image sceneImage = await scene.toImage(100, 100);
       expect(sceneImage, isA<CkImage>());
     });
-  // TODO: https://github.com/flutter/flutter/issues/60040
+
+    test('pushColorFilter does not throw', () async {
```

Comment:
Bonus points for making it a screenshot test. To do that, simply rename the file to `scene_golden_test.dart` and use golden_tester.dart like normal ([example](https://github.com/flutter/engine/blob/6ae2594b977eb2e7a32154c88399a37d7a7a2f92/lib/web_ui/test/canvaskit/canvas_golden_test.dart#L25)).

Reply:
although the test would need to be skipped outside chrome.